ci/azp-pipeline.yml

```diff
@@ -14,8 +14,8 @@ pr:
 
 variables:
   GOPATH: $(Agent.BuildDirectory)/go
-  PATH: /bin:/usr/bin:/sbin:/usr/sbin:/usr/local/bin:/usr/local/sbin
-  GOVER: 1.12.5
+  PATH: $(Agent.BuildDirectory)/go/bin:/bin:/usr/bin:/sbin:/usr/sbin:/usr/local/bin:/usr/local/sbin
+  GOVER: 1.11.5
```

Comment:
I don't remember this change last time. Looks like ci.properties was coded to use 1.12.5.